core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161115_pending_workspace_access.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="pending-ws-access-1">
+        <sql stripComments="true">
```

Comment:
No need to change if it's generally accepted to do it this way. I think the only reason to prefer using the Liquibase model is to minimize the amount of rewriting if we switched to a different database engine. You do still have to be careful, but Liquibase can do some of the translation to different dialects.

In this case, I think the biggest gain (which could probably also be done with straight SQL) would be pulling the foreign key constraints into a the `create table` statement so that it's all part of one atomic change set. May not be worth changing at this point (it's a pain to test) but something to think about for next time.

It might be worth a larger discussion about which style we want to prefer.